genkeymap/genkeymap.c

```diff
@@ -136,7 +136,7 @@ int main(int argc, char **argv)
         fprintf(outf, "[%s]\n", sections[idx]);
         e.state = states[idx];
 
-        for (i = 8; i <= 137; i++) /* Keycodes */
+        for (i = 8; i < 137; i++) /* Keycodes */
         {
             if (is_evdev)
                 e.keycode = xfree86_to_evdev[i-8];
```

Comment:
I'm not sure about this line.
xfree86_to_evdev have 129 elements [0,128] but with `i <= 137` it can try access index 129